src/templates/styles.module.scss

```diff
@@ -189,4 +201,76 @@
   text-align:center;
 }
 
+/**
+ * team styling
+ */
+
+ .team_heading {
+   margin-top: 5rem;
+   display: block;
+ }
+
+ .team{
+   display:flex;
+   flex-wrap: wrap;
+   flex:60%;
+   max-width:75%;
+  }
+  .member {
+    flex: 0 0 auto;
+    min-width: 13rem;
+    margin: 3px;
+    font-size: 1rem;
+    padding: 1.5rem;
+  }
+
+  .member_name {
+    line-height: 3rem;
+  }
+
+  .twitter_button {
+    border: 2px inset $black;
+    padding: .4rem;
+  }
+
+  .twitter_icon {
+    margin-bottom: -.2rem;
+    width:1rem;
+  }
+
+  .team_avatar{
+      width: 4rem;
+      height: 4rem;
+      margin: 1em auto;
+      background-size: cover;
+      background-repeat: no-repeat;
+      background-position: center center;
+      -webkit-border-radius: 99rem;
```

Comment:
no need for prefixes, we have autoprefixer set up

Reply:
That's cool, I didn't know. removed it. Thanks!!